### ISSUE TITLE
Final release 2.0.0 

### ISSUE THREAD

**Issue by mxochicale** [SciKit-Surgery/scikit-surgeryutils#57]
I am opening this issue to plan final release 2.0.0 from the release candidate [v2.0rc0](https://github.com/SciKit-Surgery/scikit-surgeryutils/releases/tag/v2.0rc0)   https://pypi.org/project/scikit-surgeryutils/2.0rc0/ 
* [x] Review issues (check status, decide if we address them or leave them for future work, etc)
* [x] https://github.com/SciKit-Surgery/scikit-surgeryutils/issues/56
* [x] Updates ci.yml for QT settings  and Qt for X11 Requirements 
* [x]  https://github.com/SciKit-Surgery/scikit-surgeryutils/issues/58
* [x] Update classifiers in setup.py https://pypi.org/classifiers/
* [x] Update settings (branches, `.readthedocs.yaml`) in https://readthedocs.org/projects/scikit-surgeryutils/builds/
* [x] Fix docs removed "build succeeded, 50 warnings." remotely https://readthedocs.org/projects/scikit-surgeryutils/builds/21077788/ but not locally. 
* [x] not solve but added few comments if we decide to address it https://github.com/SciKit-Surgery/scikit-surgeryutils/issues/49
* [x] solved https://github.com/SciKit-Surgery/scikit-surgeryutils/issues/54
* [x] Add list of unit tests locally and remotely for python versions 3.8, 3.9, 3.10, 3.11 and multi-os (ubuntu, mac, and windows)
  Passed local and remote tests for python versions 3.8, 3.9, 3.10, 3.11. This is a summary for python 3.8:
  Local OS Ubuntu 22.04.1 LTS; Kernel; Linux 5.19.0-35-generic (19 passed, 7 skipped in 14.66s)
  CI ubuntu-latest, 3.8 ([23 passed, 3 skipped in 14.34s](https://github.com/SciKit-Surgery/scikit-surgeryutils/actions/runs/5337235979/jobs/9672996857#step:6:848))
  CI windows-latest, 3.8 ([24 passed, 2 skipped in 22.22s ](https://github.com/SciKit-Surgery/scikit-surgeryutils/actions/runs/5337235979/jobs/9672996958#step:7:195))
  CI macos-latest, 3.8 ( [24 passed, 2 skipped in 65.46s (0:01:05)](https://github.com/SciKit-Surgery/scikit-surgeryutils/actions/runs/5337235979/jobs/9672997063#step:7:170 ) ) 
* [x] Summary of skipped test in   Local OS Ubuntu 22.04.1 LTS; Kernel; Linux 5.19.0-35-generic 
  tests/test_common_overlay_apps.py::test_OverlayOnVideoFeedCropRecord_from_file SKIPPED
  tests/test_common_overlay_apps.py::test_OverlayOnVideoFeedCropRecord_from_webcam SKIPPED
  tests/test_common_overlay_apps.py::test_OverlayBaseWidgetRaisesNotImplementedError SKIPPED
  tests/test_common_overlay_apps.py::test_OverlayOnVideoFeedCropRecord_set_roi SKIPPED
  tests/test_rendermodelslikecamera.py::test_rendermodelslikecamera SKIPPED
  tests/test_sksurgerytextoverlay.py::test_sksurgerytextoverlay SKIPPED
  tests/utils/test_image_utils.py::test_image_to_pixel_valid_rgb_example SKIPPED 
* [x] Publish pypi final release 2.0.0
```
git tag -a 'v2.0.0' -m "Release version v2.0.0"
git push origin v2.0.0
```
* [x] Add release notes as in https://github.com/SciKit-Surgery/scikit-surgeryvtk/issues/197

Thanks

**Comment by mxochicale** [comment]
tox logs

> FAILED tests/test_sksurgeryarucotest.py::test_sksurgeryarucotest - AttributeError: module 'cv2.aruco' has no attribute 'Dictionary_get'

SOLUTION: https://stackoverflow.com/questions/74964527/attributeerror-module-cv2-aruco-has-no-attribute-dictionary-get

package versions
>#- opencv-contrib-python-headless>=4.7.0.72 #https://pypi.org/project/opencv-contrib-python-headless/#history
>     - opencv-contrib-python-headless>=4.6.0.66 #https://pypi.org/project/opencv-contrib-python-headless/#history

**Comment by mxochicale** [comment]
Removing package version for "opencv-contrib-python-headless" and "scikit-surgeryarucotracker" fix installation of requirements in python3.8 for ubuntu-latest and macos-latest but not for windows-latest and others python versions: https://github.com/SciKit-Surgery/scikit-surgeryutils/actions/runs/5255757370

**Comment by mxochicale** [comment]
### E       AttributeError: module 'cv2.aruco' has no attribute 'Dictionary_get'

:red_circle:  Trying to remotely use both opencv-python and opencv-contrib-python-headless but don't fix the issue 
https://github.com/SciKit-Surgery/scikit-surgeryutils/actions/runs/5265407978/jobs/9517990501#step:6:850

:red_circle:   Trying to remotely use opencv-contrib-python only but don't fix it > https://github.com/SciKit-Surgery/scikit-surgeryutils/actions/runs/5265593578/jobs/9518429427#step:6:858

:red_circle:  Trying to remotely use both opencv-python and opencv-contrib-python but don't fix the issue 
https://github.com/SciKit-Surgery/scikit-surgeryutils/actions/runs/5265666785/jobs/9518605336#step:6:852

**Comment by mxochicale** [comment]
* [x] Created release 2.0.0
git tag -a 'v2.0.0' -m "Release version v2.0.0"
git push origin v2.0.0
* [x] Published pypi final release 2.0.0 https://pypi.org/project/scikit-surgeryutils/2.0.0/
* [x] Manually added final release notes > https://github.com/SciKit-Surgery/scikit-surgeryutils/releases/tag/v2.0.0
